src/components/Scheme/SchemeContainer.tsx

```diff
@@ -97,34 +139,45 @@ class SchemeContainer extends React.Component<IProps, null> {
 
         <Sticky enabled top={50} innerZ={10000}>
           <h2 className={css.schemeName}>
-            {schemeName(schemeState, schemeState.address)}
+            {schemeFriendlyName}
           </h2>
 
           <div className={css.schemeMenu}>
             <Link className={proposalsTabClass} to={`/dao/${daoAvatarAddress}/scheme/${schemeId}/proposals/`}>Proposals</Link>
             <TrainingTooltip placement="top" overlay={"Learn about the protocol parameters for this scheme"}>
-              <Link className={infoTabClass} to={`/dao/${daoAvatarAddress}/scheme/${schemeId}/info/`}>Info</Link>
+              <Link className={infoTabClass} to={`/dao/${daoAvatarAddress}/scheme/${schemeId}/info/`}>Information</Link>
             </TrainingTooltip>
+            {
+              this.state.crxRewarderProps ?
+                <TrainingTooltip placement="top" overlay={this.state.crxRewarderProps.shortDescription}>
+                  <Link className={crxTabClass} to={`/dao/${daoAvatarAddress}/scheme/${schemeId}/crx/`}>{this.state.crxRewarderProps.friendlyName} ({approvedProposals.length})</Link>
+                </TrainingTooltip>
+                : ""
+            }
+          </div>
+          <div className={css.createProposal}>
             <TrainingTooltip placement="topRight" overlay={"A small amount of ETH is necessary to submit a proposal in order to pay gas costs"}>
               <a className={
                 classNames({
-                  [css.createProposal]: true,
                   [css.disabled]: !isActive,
                 })}
               data-test-id="createProposal"
-              href="javascript:void(0)"
+              href="#!"
```

Comment:
this is new to me... cool!

Reply:
yeah, this gets rid of the runtime console warnings